src/components/carousel/DotsPaginationButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface DotsButtonProps {
+  onClick: () => void;
+  isCurrent: boolean;
+  currentDot: number;
+  totalDots: number;
+}
+
+const DotsPaginationButton: FC<DotsButtonProps> = ({
+  onClick,
+  isCurrent,
+  currentDot,
+  totalDots,
+}) => {
+  const {
+    dotsPaginationButton,
+    dotsPaginationButtonActive,
+    dotsPaginationButtonWrapper,
+  } = useMultiStyleConfig('Carousel');
+
+  return (
+    <chakra.button
+      __css={dotsPaginationButtonWrapper}
+      onClick={onClick}
+      aria-current={isCurrent}
+      aria-label={`slide indicator ${currentDot} of ${totalDots}`}
```

Comment:
this code is identical with the NumbersPaginationButton. You could share it and try to leverage the variant feature from Chakra CSS. We already have a variant for fullScreen in the carousel so you can check how it's done there